source/upgrade/kolla.rst

```diff
@@ -278,3 +340,103 @@ HAProxy
      {% endfor %}
 
      # customer specific configuration
+
+Running the upgrade
+===================
+
+Common
+------
+
+.. code-block:: console
+
+   osism-kolla upgrade common
+
+HAProxy
+-------
+
+.. code-block:: console
+
+   osism-kolla upgrade haproxy
+
+Logging
+-------
+
+.. code-block:: console
+
+   osism-kolla upgrade elasticsearch
+   osism-kolla upgrade kibana
+
+Infrastructure
+--------------
+
+.. code-block:: console
+
+   osism-kolla upgrade memcached
+   osism-kolla upgrade mariadb
+   osism-kolla upgrade rabbitmq
+   osism-kolla upgrade redis
+   osism-kolla upgrade openvswitch
+
+Storage (optional)
+------------------
+
+.. code-block:: console
+
+   osism-kolla upgrade iscsi
+   osism-kolla upgrade multipath
+
+Openstack services
+------------------
+
+.. code-block:: console
+
+   osism-kolla upgrade keystone
+   osism-kolla upgrade horizon
+   osism-kolla upgrade glance
+   osism-kolla upgrade cinder
+   osism-kolla upgrade neutron
+   osism-kolla upgrade heat
+
+Nova
+----
+
+* Upgrade nova on the controller nodes first:
+
+.. code-block:: console
+
+   osism-kolla upgrade nova -l controller
+
+* Upgrade nova on the compute nodes:
+
+.. code-block:: console
+
+   osism-kolla upgrade nova -l compute
+
```

Comment:
I think it makes sense to add a note that other services may be available in an environment.